docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_4.cs

```diff
@@ -8,7 +8,7 @@ static void Main()
             // Query.
             IEnumerable<string> query =
                 from rad in radii
-                select String.Format("Area = {0}", (rad * rad) * 3.14);
+                select $"Area = {(rad * rad) * 3.14}";
```

Comment:
nit picking from math: replace `3.14` with `System.Math.PI`? (or with adding `using System;`)

Reply:
@pkulikov If I make the change, then I'll also need to update the expected result (in the bottom of the file). Will it be ok to format the `area` to two decimals in the text, if I use `Math.PI`?

Reply:
@mikkelbu that's a good point.

I've just checked: if the output is formatted to two decimals:
```csharp
select $"Area = {rad * rad * Math.PI:F2}";
```

the output slightly changes:
```
Area = 3.14
Area = 12.57
Area = 28.27
```